plugin/src/test/java/io/jenkins/plugins/casc/yaml/OverrideMergeStrategyTest.java

```diff
@@ -0,0 +1,67 @@
+package io.jenkins.plugins.casc.yaml;
+
+import io.jenkins.plugins.casc.ConfigurationAsCode;
+import io.jenkins.plugins.casc.ConfigurationContext;
+import io.jenkins.plugins.casc.ConfiguratorException;
+import io.jenkins.plugins.casc.ConfiguratorRegistry;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.contrib.java.lang.system.EnvironmentVariables;
+import org.junit.rules.ExpectedException;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThrows;
+
+public class OverrideMergeStrategyTest {
+
+    @ClassRule
+    public static JenkinsRule j = new JenkinsRule();
+
+    private static ConfigurationContext context;
+
+    @ClassRule
+    public static final EnvironmentVariables environment = new EnvironmentVariables();
+
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    @BeforeClass
+    public static void setUp() {
+        environment.set("CASC_MERGE_STRATEGY", "override");
```

Comment:
aren't we missing a test for overriding the strategy via config file?

Reply:
Do you mean overriding it via the CasC file? How can I get the value if there are multiple files which have different name?

Reply:
yeah, shouldn't this be set automatically by the SelfConfigurator? (serially) 

I'm not 100% sure @jetersen seemed to be suggesting it could be done that way

Reply:
IMO, the strategy setting is above the CasC file. We need the strategy name before trying to merging them.

Reply:
I think not a blocker anyway if you can resolve the other issues, we should be able to ship